src/main/scala/com/atomist/tree/marshal/LinkedJsonTreeDeserializer.scala

```diff
@@ -103,6 +102,21 @@ object LinkedJsonTreeDeserializer extends LazyLogging {
     }
 }
 
+class EmptyLinkableContainerTreeNode() extends ContainerTreeNode {
+
+  override def nodeName: String = "empty"
+
+  override def value: String = ""
```

Comment:
I kind of feel that in Scala, things that can be null should be Optional. But I suppose changing this will have bigger implications. Null would be better than empty string IMO. IF things barf on null, then they should be fixed too :)